EIPS/eip-1.md

```diff
@@ -168,37 +246,45 @@ if neither the email address nor the GitHub username are given.
 
-### `type` header
+#### `type` header
 
 The `type` header specifies the type of EIP: Standards Track, Meta, or Informational. If the track is Standards please include the subcategory (core, networking, interface, or ERC).
 
-### `category` header
+#### `category` header
 
 The `category` header specifies the EIP's category. This is required for standards-track EIPs only.
 
-### `created` header
+#### `created` header
 
 The `created` header records the date that the EIP was assigned a number. Both headers should be in yyyy-mm-dd format, e.g. 2001-08-14.
 
-### `requires` header
+#### `requires` header
 
 EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on. If such a dependency exists, this field is required.
 
 A `requires` dependency is created when the current EIP cannot be understood or implemented without a concept or technical element from another EIP. Merely mentioning another EIP does not necessarily create such a dependency.
 
-## Linking to External Resources
+### Linking to External Resources
 
 Other than the specific exceptions listed below, links to external resources **SHOULD NOT** be included. External resources may disappear, move, or change unexpectedly.
 
 The process governing permitted external resources is described in [EIP-5757](./eip-5757.md).
 
-### Execution Client Specifications
+External resources not so permitted **MAY** be included *at the Editors' discretion*.
+
+A complete reference to every external resource **MUST** be included in a Normative or Informative References section.
+
+We take several precautions to ensure the continued accessibility of all external resources.  First, we follow the requirements of the [RFC 7322 RFC Style Guide](https://www.rfc-editor.org/rfc/rfc7322.html): references **MUST** be full citations, including author(s), title, and publication information -- URLs are acceptable, but **MUST NOT** be the sole information provided for a reference.  In addition, Links **SHOULD** meet the Requirements for Origins in [EIP-5757](./eip-5757.md), and a Digital Object Identifier (DOI), where available, **MUST** be included. Within "fair use", short quotations from referenced works are encouraged, as embedded quotations will not change. 
+
+According to the IESG Statement on [Normative and Informative References](https://www.ietf.org/about/groups/iesg/statements/normative-informative-references/), Normative references "specify documents that must be read to understand or implement the technology" whereas an Informative reference "only provides additional information".  The Specification section of an EIP **MUST NOT** contain Informative references, and **SHOULD** contain all necessary Normative references. Motivation and Rationale sections **MAY** include Informative references.
```

Comment:
We do already have this distinction for EIPs (the `requires` preamble field.)

If the Specification section can only contain normative references, then I think we can infer that references in that section are normative. Similarly, if normative references are not permitted in non-Specification sections, we can infer that all references in those section are informative.

My point being: why have specific sections for normative and informative references? We can let authors use normal markdown links (and `csl-json` footnote blobs) to create references, and their location within the document decides whether they're normative or informative.

Reply:
We don't need them, you are right.  I took the one part from IETF before I wrote this part and didn't notice that I'd made it obsolete.   (And thanks for all your reviews here!)